operator/controllers/lokistack_controller.go

```diff
@@ -105,15 +105,16 @@ func (r *LokiStackReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 		return ctrl.Result{}, nil
 	}
 
-	err = handlers.CreateOrUpdateLokiStack(ctx, req, r.Client, r.Scheme, r.Flags)
+	var degraded *status.DegradedCondition
+	degraded, err = handlers.CreateOrUpdateLokiStack(ctx, req, r.Client, r.Scheme, r.Flags)
```

Comment:
```suggestion
	degraded, err := handlers.CreateOrUpdateLokiStack(ctx, req, r.Client, r.Scheme, r.Flags)
```

Reply:
Since `err` is already defined before hence I can't use `:` as you suggesting. I need to create a `degraded` var or define `err` with some new variable name.

Reply:
As long as there is at least one undefined value on the left-hand side you can still use `:=`, so this change should be fine.